### PR TITLE
tiup-playground: add info about TiProxy

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v9.0 (TiDB 9.0 versions)
- [x] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/19357
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-playground.md [hunk 1]
```diff
@@ -133,6 +133,29 @@ You can specify a `pid` in the `tiup playground scale-in` command to scale in th
...
+
+```shell
+tiup playground v8.5.0 --tiproxy 1
+```
+
+The following are command-line flags of the playground component for TiProxy:
+
+```bash
+Flags:
+      --tiproxy.binpath string       TiProxy instance binary path
```
Comment (qiancai) [inline tiup/tiup-playground.md:150]:
```suggestion
      --tiproxy int                          Specifies the number of TiProxy nodes in the cluster. If not specified, TiProxy is not deployed.
      --tiproxy.binpath string       TiProxy instance binary path
```

File: tiup/tiup-playground.md [hunk 2]
```diff
@@ -133,6 +133,30 @@ You can specify a `pid` in the `tiup playground scale-in` command to scale in th
 tiup playground scale-in --pid 86526
 ```
 
+## Deploy TiProxy
+
+[TiProxy](/tiproxy/tiproxy-overview.md) is the official proxy component of PingCAP. It is placed between the client and the TiDB server to provide load balancing, connection persistence, service discovery, and other features for TiDB.
```
Comment (lilin90) [inline tiup/tiup-playground.md:138]:
```suggestion
[TiProxy](/tiproxy/tiproxy-overview.md) is the official proxy component from PingCAP, placed between the client and the TiDB server to provide load balancing, connection persistence, service discovery, and other features for TiDB.
```

File: tiup/tiup-playground.md [hunk 3]
```diff
@@ -133,6 +133,30 @@ You can specify a `pid` in the `tiup playground scale-in` command to scale in th
...
+
+You can deploy the TiProxy for your cluster using TiUP Playground as in the following example:
```
Comment (lilin90) [inline tiup/tiup-playground.md:140]:
```suggestion
You can deploy TiProxy for your cluster using TiUP Playground, as in the following example:
```

File: tiup/tiup-playground.md [hunk 4]
```diff
@@ -133,6 +133,30 @@ You can specify a `pid` in the `tiup playground scale-in` command to scale in th
...
+
+```shell
+tiup playground v8.5.0 --tiproxy 1
+```
+
+The following are command-line flags of the playground component for TiProxy:
```
Comment (lilin90) [inline tiup/tiup-playground.md:146]:
```suggestion
TiProxy-related command-line flags in the playground component are as follows:
```

File: tiup/tiup-playground.md [hunk 5]
```diff
@@ -133,6 +133,30 @@ You can specify a `pid` in the `tiup playground scale-in` command to scale in th
...
+
+```bash
+Flags:
+      --tiproxy int                  Specifies the number of TiProxy nodes in the cluster. If not specified, TiProxy is not deployed.
+      --tiproxy.binpath string       TiProxy instance binary path
+      --tiproxy.config string        TiProxy instance configuration file
+      --tiproxy.host host            Playground TiProxy host. If not provided, TiProxy will still use host flag as its host
+      --tiproxy.port int             Playground TiProxy port. If not provided, TiProxy will use 6000 as its port
+      --tiproxy.timeout int          TiProxy max wait time in seconds for starting. 0 means no limit (default 60)
```
Comment (lilin90) [inline tiup/tiup-playground.md:155]:
```suggestion
      --tiproxy int                  Specifies the number of TiProxy nodes in the cluster. If not specified, TiProxy is not deployed.
      --tiproxy.binpath string       TiProxy instance binary path.
      --tiproxy.config string        TiProxy instance configuration file.
      --tiproxy.host host            Playground TiProxy host. If not provided, TiProxy will still use host flag as its host.
      --tiproxy.port int             Playground TiProxy port. If not provided, TiProxy will use 6000 as its port.
      --tiproxy.timeout int          TiProxy max wait time in seconds for starting. 0 means no limit (default 60).
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by djshow832 — APPROVED** [review]

File: tiup/tiup-playground.md [hunk 6]
```diff
@@ -133,6 +133,30 @@ You can specify a `pid` in the `tiup playground scale-in` command to scale in th
...
+```
+
+For more information about TiProxy deployment and usage, see [TiProxy documentation](/tiproxy/tiproxy-overview.md#installation-and-usage).
+
+In the playground component, TiProxy-related command-line flags are as follows:
+
+```bash
+Flags:
+      --tiproxy int                  Specifies the number of TiProxy nodes in the cluster. If not specified, TiProxy is not deployed.
+      --tiproxy.binpath string       TiProxy instance binary path.
```
Comment (qiancai) [inline tiup/tiup-playground.md:153]:
```suggestion
      --tiproxy int                  The number of TiProxy nodes in the cluster. If not specified, TiProxy is not deployed.
      --tiproxy.binpath string       TiProxy instance binary path.
```

**Review by lilin90 — COMMENTED** [review]
LGTM

File: tiup/tiup-playground.md [hunk 7]
```diff
@@ -133,6 +133,30 @@ You can specify a `pid` in the `tiup playground scale-in` command to scale in th
...
+In the playground component, TiProxy-related command-line flags are as follows:
+
+```bash
+Flags:
+      --tiproxy int                  The number of TiProxy nodes in the cluster. If not specified, TiProxy is not deployed.
+      --tiproxy.binpath string       TiProxy instance binary path.
+      --tiproxy.config string        TiProxy instance configuration file.
+      --tiproxy.host host            Playground TiProxy host. If not provided, TiProxy will still use host flag as its host.
+      --tiproxy.port int             Playground TiProxy port. If not provided, TiProxy will use 6000 as its port.
+      --tiproxy.timeout int          TiProxy max wait time in seconds for starting. 0 means no limit (default 60).
```
Comment (qiancai) [inline tiup/tiup-playground.md:157]:
```suggestion
      --tiproxy.timeout int          TiProxy maximum wait time in seconds for starting. 0 means no limit (default 60).
```

**Pull request merged**